src/View/ProgressWindow.py

```diff
@@ -56,6 +58,17 @@ def update_progress(self, progress_update):
         self.text_field.setText(progress_update[0])
         self.progress_bar.setValue(progress_update[1])
 
+    def prompt_calc_dvh(self):
+        choice = QMessageBox.question(self, "Calculate DVHs?", "RTSTRUCT and RTDOSE datasets identified. Would you "
+                                                               "like you calculate DVHs? (This may take up to several "
```

Comment:
change from:
Would you like you calculate DVHs
to:
Would you like to calculate DVHs